### ISSUE TITLE
LJ Assertion failed

### ISSUE THREAD

**Issue by olegrok** [tarantool/tarantool#4053]
I use Tarantool 2.1.1-185-gf583b6c82 on macOS Mojave

I got an error:
```
tarantool> jp.query(data, "$..feature_codes[*][?(@.base_feature_code='CPKG')]")
Assertion failed: (J->baseslot >= 1+1 && J->baseslot < 250), function rec_check_slots, file lj_record.c, line 90.
[1]    26197 abort      tarantool
```

Now I can't reproduce this problem, but I share console log with @kyukhin

**Comment by kyukhin** [comment]
We need a reproducer anyway.

**Comment by kyukhin** [comment]
No reproducer for more than month, closing.

**Comment by olegrok** [comment]
May be related with https://github.com/openresty/luajit2/commit/1951d9a4065cb84738d9077c1a77475cc81b4a6d

Reopen issue, I think we should cherry-pick this fix.

**Comment by kyukhin** [comment]
Closed by bfde5242b65dd4d972c242e7c21673386975bcd0
